src/KafkaFlow.Retry/Durable/Polling/QueueTracker.cs

```diff
@@ -68,11 +65,11 @@ internal void ScheduleJobs(CancellationToken cancellationToken = default)
                     continue;
                 }
 
-                this.ScheduleJob(jobDataProvider, cancellationToken);
+                this.ScheduleJobAsync(jobDataProvider, cancellationToken);
```

Comment:
Please await this call and apply also `ConfigureAwait(false)`.

Reply:
oh no. I missed this. Added.

Reply:
Thank you Fernando 👌